ergoCubSN000/ergocub_head.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!DOCTYPE robot PUBLIC "-//YARP//DTD yarprobotinterface 3.0//EN" "http://www.yarp.it/DTD/yarprobotinterfaceV3.0.dtd">
+
+<robot name="ergoCubSN000" build="0" portprefix="/ergocub" xmlns:xi="http://www.w3.org/2001/XInclude">
+
+  <params>
+    <xi:include href="hardware/electronics/pc104.xml" />
+  </params>
+
+  <devices>
+    <!-- motor controllers wrappers and remappers -->
+    <xi:include href="wrappers/motorControl/head-mc_wrapper.xml" />
+    <xi:include href="wrappers/motorControl/head-mc_remapper.xml" />
+
+    <!-- Head -->
+    <xi:include href="hardware/motorControl/head-eb20-j0_1-mc.xml" />
+    <xi:include href="hardware/motorControl/head-eb21-j2_3-mc.xml" />
+
+    <!-- Inertials -->
+    <xi:include href="wrappers/inertials/head-inertials_wrapper.xml" />
+    <xi:include href="hardware/inertials/head-inertial.xml" />
+
+    <xi:include href="calibrators/head-calib.xml" />
+  </devices>
+
+</robot>
```

Comment:
This file needs to be removed in favor of `head.xml`.
Also, it may be worth merging the content.

Reply:
done!

Reply:
Super! Thanks!